ostelco-ios-client/ViewControllers/Home/HomeViewController.swift

```diff
@@ -49,6 +49,7 @@ class HomeViewController: ApplePayViewController {
 
     private func showWelcomeMessage() {
         if HomeViewController.newSubscriber {
+            HomeViewController.newSubscriber = false
```

Comment:
This is not persisted anywhere, so any time the user restarts the device, they'll still see the welcome message. Is that the behavior you're looking for?

Reply:
Thank you for the clarification!